tests/integration/introduction/test_introduction.py

```diff
@@ -9,7 +9,7 @@ def test_mail_link_contains_ru_ref_in_subject(self):
 
         # When on the landing page
         # Then the email link is present with the ru_ref in the subject
-        self.assertRegexPage('\"mailto\:.+\?subject\=.+123456789012A\"')
+        self.assertRegexPage(r'\"mailto\:.+\?subject\=.+123456789012A\"')
```

Comment:
Is this change related to this PR?

Reply:
Linting error blocked build. Could be committed elsewhere, sure.